app/src/main/java/com/globant/counter/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.globant.counter.mvp.model
+
+import com.globant.counter.utils.MathUtils
+
+class CalculatorModel {
+
+    lateinit var displayContent: String
+    var operator: String = ""
+    var isResult: Boolean = true
+
+    fun makeOperation(firstNumber: Double, secondNumber: Double): Double {
+        val result = MathUtils.makeOperation(firstNumber, operator, secondNumber)
+        displayContent = result.toString()
+        clearOperator()
+        return result
+    }
+
+    private fun clearOperator() {
+        operator = ""
```

Comment:
@tomaskovacs  this is a "magic number". As a rule in royal project, it is enforced that if you create a constant for values that you want to set or compare to variables. In the project we already have an empty string constant for example, so please create for this